packaging/msi/SumoLogic.wixext/SumoLogic.wixext/Config.cs

```diff
@@ -8,6 +8,10 @@ public class Config
     {
         public string InstallationToken { get; set; }
         public Dictionary<string, string> CollectorFields { get; set; }
+        public bool RemotelyManaged { get; set; }

```

Comment:
Do we need to make any doc updates to mention these options
https://github.com/SumoLogic/sumologic-otel-collector/blob/main/docs/installation.md#installation

Reply:
Opening a PR in https://github.com/SumoLogic/sumologic-documentation for the changes.